### PR TITLE
Update USERS.md

### PR DESCRIPTION
**Pull request by imathu**
add Cilium user "PostFinance AG"

Please ensure your pull request adheres to the following guidelines:

- [x] For first time contributors, read [Submitting a pull request](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/#submitting-a-pull-request)
- [ ] All code is covered by unit and/or runtime tests where feasible.
- [ ] All commits contain a well written commit description including a title,
      description and a `Fixes: #XXX` line if the commit addresses a particular
      GitHub issue.
- [ ] All commits are signed off. See the section [Developer’s Certificate of Origin](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/#dev-coo)
- [ ] Provide a title or release-note blurb suitable for the release notes.
- [ ] Thanks for contributing!

Fixes: #issue-number

```release-note
Add "PostFinance AG" to USERS.md
```

### CODE REVIEW

**Comment by kaworu** [comment]
Hi @imathu, thanks for the PR! Could you [Sign-Off the commit](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/#developer-s-certificate-of-origin) please?

File: USERS.md [hunk 1]
```diff
@@ -108,6 +108,12 @@ Users (Alphabetically)
       U: ENI networking, L3/L4 policies, FQDN based policy, FQDN filtering
       Q: ungureanuvladvictor
       
+    * N: PostFinance AG 
+      D: PostFinance is using Cilium as their CNI for all mission critical, on premise k8s clusters 
+      U: Networking, network policies, kube-proxy replacement 
+      L: https://github.com/postfinance
+
+
```
Comment (qmonnet) [inline USERS.md]: Please would you mind removing the second blank line, to keep the style consistent with the rest of the file?

**Review by imathu — COMMENTED** [review]
additional space removed

**Comment by qmonnet** [comment]
Re-opened as #14831.

**Pull request closed without merging**